notebooks/HASP/COSLifetimePositions/COSLifetimePositions.ipynb

```diff
@@ -0,0 +1,965 @@
+    "\n",
+    "In this notebook, we will compare the COS LSFs from multiple CENWAVEs and lifetime positions to better understand these effects.  Then we will consider three ways to deal with them.  The first is not to use the final HASP and HSLA products at all, but to create products that contain data from a single grating, LP, and CENWAVE and fit them separately.  The second is to combine these single-configuration products into a co-added spectrum, first smoothing the higher-resolution spectra to the resolution of the lowest-resolution spectrum.  The third approach is to use the HASP/HSLA final products, but to scale and combine the relevant LSFs to match your data.\n",
+    "\n",
+    "By the way, the G130M and G160M gratings have different LSFs in the region of overlap, so everything in this notebook applies to the case of multiple gratings.  The difference is that neither HASP nor HSLA co-adds data from different instruments or gratings, so the final data products are immune to this effect."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2cbdc832",
+   "metadata": {},
+   "source": [
+    "### Imports\n",
+    "\n",
+    "We will be using multiple libraries to retrieve and analyze data: \n",
+    "* `astropy.units` to assign units to various quantities\n",
+    "* `astropy.convolution.convolve` to convolve functions with a Gaussian\n",
+    "* `astropy.io.ascii` and `fits` to work with ASCII and FITS files\n",
+    "* `astropy.visualization.quantity_support` to put units on our plots\n",
+    "* `astroquery.mast.Observations` to retrieve COS spectra from MAST\n",
+    "* `glob` to work with multiple files in our directories\n",
+    "* `matplotlib.pyplot` to plot spectra\n",
+    "* `numpy` to calculate various values\n",
+    "* `os` to interact with the operating system\n",
+    "* `pathlib.Path` to create a data directory \n",
+    "* `scipy.optimize.curve_fit` to fit a function to a spectrum\n",
+    "* `shutil` to perform directory and file operations\n",
+    "* `specutils.Spectrum1d` to create Spectrum1d objects\n",
+    "\n",
+    "We recommend creating a HASP-specific `conda` environment. You can check out our [Setup.ipynb](https://github.com/spacetelescope/hst_notebooks/tree/main/notebooks/HASP/Setup) notebook to create such an environment. Alternatively, you can also download the required dependencies to run this notebook with the terminal command: \n",
+    "\n",
+    "`pip install -r requirements.txt`\n",
```

Comment:
leave out this part - installing requirements.txt does not do the same thing as running that setup notebook, it only has jupyter notebook package versions specified.